initramfs-init.in

```diff
@@ -381,6 +383,31 @@ fi
 # zpool reports /dev/zfs missing if it can't read /etc/mtab
 ln -s /proc/mounts /etc/mtab
 
+if echo $@ | grep -q "dasd" || echo $@ | grep -q "qeth" ; then
```

Comment:
This is _very_ bad approach. It may fail when `$1` is happened to be something like `-e`, `-n`, … `$@` should be quoted, otherwise it wouldn’t make much sense to use `$@` instead of `$*`.

What is `dasd` and `qeth`? Do you want to find exact match only or any string that contains one of these as a substring?